packages/kbn-rule-data-utils/index.ts

```diff
@@ -7,6 +7,7 @@
  */
 
 export * from './src/default_alerts_as_data';
+export * from './src/legacy_alerts_as_data';
```

Comment:
`default_alerts_as_data` contains the fields that end up in the `.alerts-framework-mappings` component template

`legacy_alerts_as_data` contains the fields that are in the `.alerts-technical-mappings` from the rule registry but are not in `.alerts-framework-mappings`